src/components/Layout/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, ReactNode } from 'react';
+
+import Head from 'next/head';
+import Navbar from '@/components/Navbar';
+import Footer from '@/components/Footer';
+
+type LayoutProps = {
+    title: string;
+    children: ReactNode;
+};
+
+const Layout: FC<LayoutProps> = ({ title, children }) => {
+    return (
+        <div>
```

Comment:
- Tip: Instead of wrapping this in a div we can also wrap it in a react fragment. 
```js
return (
    <>
        <Head>
             .....
    </>
)
```

Reply:
updated